packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```diff
@@ -233,6 +233,49 @@ Resources:
               - Ref: AWS::NoValue
       RoleName:
         Fn::Sub: cdk-${Qualifier}-image-publishing-role-${AWS::AccountId}-${AWS::Region}
+  LookupRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Action: sts:AssumeRole
+            Effect: Allow
+            Principal:
+              AWS:
+                Ref: AWS::AccountId
+          - Fn::If:
+              - HasTrustedAccounts
```

Comment:
Will we also get `HasLookupTrustedAccounts` ?

Reply:
Yes.

Reply:
This is a very limited set of things that can be looked up.. Why so restricted?

Reply:
Why not make this use ReadOnlyAccess